src/script/lua_api/l_env.cpp

```diff
@@ -78,7 +80,9 @@ void LuaABM::trigger(ServerEnvironment *env, v3s16 p, MapNode n,
 	lua_pushnumber(L, active_object_count);
 	lua_pushnumber(L, active_object_count_wider);
 
-	PCALL_RESL(L, lua_pcall(L, 4, 0, errorhandler));
+	int result = lua_pcall(L, 4, 0, errorhandler);
+	if (result)
+		scriptIface->scriptError(result, "LuaABM::trigger");
```

Comment:
Because:
- trigger on its own is too vague, since __FUNCTION__ only includes the function name and not the class
- this uses ScriptApiBase::scriptError() is being used externally so the macro PCALL_RES() won't work.  I could write __FUNCTION__ here, but what's the point if this exact piece of code is being used in exactly one function??